pkg/models/model.go

```diff
@@ -18,8 +18,8 @@ import (
 	"reflect"
 	"strings"
 
-	"github.com/golang/protobuf/descriptor"
 	"github.com/golang/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
```

Comment:
Is it okay that "proto" package is still imported from `github.com/golang/protobuf/proto` and not from `google.golang.org/protobuf...` ?

Reply:
Yes, module `github.com/golang/protobuf` has actually switched its implementation in v1.4 to the new API under the hood. See release notes for more info: https://github.com/golang/protobuf/releases/tag/v1.4.0 

You can also see that the `github.com/golang/protobuf` v1.4 (final release for v1) actually has a dependency on the new API `google.golang.org/protobuf` v1.21 which is practically a new major version - v2.

Reply:
Thanks a lot